tutorial.md

```diff
@@ -1,101 +1,85 @@
-to 2.
+8. Under "Start and Duration" section, set "Start at" to "Manual", and "Duration" to 2.
 
-In the "Trace Option" section, click on the "Configure" button to pop-up a
-configuration window where you can select which can of performance counters you
-want to be captured.
+9. In the "Trace Option" section, click on the "Configure" button to pop-up a configuration window where you can select which can of performance counters you want to be captured.
 
-<img src="../images/system_profiler/capture-config.png" width="500px">
+    <img src="../images/system_profiler/capture-config.png" width="500px">
 
-In the "Output" section, please select an "Output Directory" where the capture
-file will be stored. The "File Name" should be auto-filled, you can edit the
-file name if need be.
+10. In the "Output" section, please select an "Output Directory" where the capture file will be stored. The "File Name" should be auto-filled. You can also edit the file name.
 
-In the GPU section, click on "Select" to bring up the counter selection dialog.
-Click on "default" to select the set of default counters and press "OK".
+11. In the GPU section, click on "Select" to bring up the counter selection dialog. Click on "default" to select the set of default counters and press "OK".
 
-<img src="../images/system_profiler/counter-config.png" width="500px">
+    <img src="../images/system_profiler/counter-config.png" width="500px">
 
-Once all this is done, press "OK". This starts the selected app on the Android
-device, and creates a pop-up window with a "Start" button. Click the "Start"
-button to start the performance capture, and wait for a couple of seconds for
+12. Once all this is done, press "OK". This starts the selected app on the Android device, and creates a pop-up window with a "Start" button. Click the "Start" button to start the performance capture, and wait for a couple of seconds for
 the trace to terminate.
 
-Once the capture is completed, click on "Open Trace". This leads to a view
-similar to [systrace](https://developer.android.com/studio/profile/systrace).
+13. Once the capture is completed, click on "Open Trace". This leads to a view similar to [systrace](https://developer.android.com/studio/profile/systrace).
 
-<img src="../images/system_profiler/system-profile.png" width="500px">
+    <img src="../images/system_profiler/system-profile.png" width="500px">
 
-Compare to systrace, there are now additional information relating to the GPU.
-The GPU Queue section shows when GPU commands are submitted and when the GPU
-actually starts work.  The GPU Counters shows various performance counters to
-help optimize the graphics works in the app.
+    Compare to systrace, there are now additional information relating to the GPU. The GPU Queue section shows when GPU commands are submitted and when the GPU actually starts work.  The GPU Counters shows various performance counters to help optimize the graphics works in the app.
```

Comment:
*Compared* to systrace, there *is* now additional...

I would phrase it differently though: "In addition to the data available in systrace, AGI also shows GPU performance information. ..." or something like that.

Reply:
See https://github.com/google/agi/pull/182/commits/62bf1b179b61c5e8ae25464df3961aa218ff1bd8